framework_tvb/tvb/adapters/simulator/simulator_adapter.py

```diff
@@ -179,6 +179,10 @@ def __init__(self):
     def get_form_class(self):
         return SimulatorAdapterForm
 
+    @staticmethod
+    def get_view_model():
```

Comment:
This should not be. Same functionality is covered by the existing `ABCAdapter.get_view_model_class` method.

Reply:
removed it since the suggestion below fixed the initial issue